README.md

```diff
@@ -1,5 +1,15 @@
-This repository contains a list of tests in the Salt Test Suite that are **not** executed. Skipping
-tests is bad and should only be used as a last resort.
+This repository contains a list of tests in the Salt Test Suite that are **not** executed.
 
-The list is formatted using the TOML markup language and skipped tests are grouped by file/test level and then by "type". Missing
-groups do  not have t be listed explicitely.
\ No newline at end of file
+The list is formatted using the TOML markup language. Skipped tests are grouped by file/test level and then by type. Missing groups do not have to be listed explicitly.
+
+The list is used by [`salt-test`](https://github.com/openSUSE/salt-test#readme), a test launcher that skips the collection of files in the `[ignore]` sections and deselects all tests in the `[skip]` section.
+
+### `ignore` vs `skip`
+
+Prefer `skip` to `ignore`.
+Reason: `ignore` skips over a whole Python module (i.e. a file). It's needed when the import of the module causes errors, e.g. `ImportError`. We don't have all of Salt's Python dependencies packaged and need to exclude files that don't guard their imports.
+`skip` has finer granularity and is prefered for that reason.
+
+### Adding new entries to `skipped_tests.toml`
+
+This repository disallows direct commits to `main` to ensure all additions to `skipped_tests.toml` are agreed upon by [openSUSE's Salt team](https://github.com/orgs/openSUSE/teams/salt). All changes must come via a pull request that is approved by at least two other maintainers. All new entries must include a comment that explains why a given test is failing. The pull request description should explain why a test case is not getting fixed and instead added to this skiplist. Pull request reviewers are asked to critically review this explanation.
```

Comment:
Maybe it makes sense to additionaly request for some comments to the test listed to skip.
I was trying to identify most common reasons of tests fails and found that at least for limited amount of the test it's better to adjust tests itself. For example: https://github.com/saltstack/salt/pull/65266. Without this fix some of the teststs are failing just because tests itself are not expecting the presence of optimized pyc files.
So we can understand better the root cause of skipping the test. If there are some chances to remove the skip for the future version it would be nice to have such refs.

Reply:
Thanks for taking a look at the test failures already! I agree that references in the comments make sense. Do you think that, even tough it's worded generic, this part covers what you're asking for? Would you like to propose an alternative wording to make it clearer?

> All new entries must include a comment that explains why a given test is failing.